omnibus/config/software/datadog-agent.rb

```diff
@@ -132,12 +132,10 @@
 
   # System-probe
   if windows_target?
-    unless windows_arch_i386?
-      if ENV['WINDOWS_DDNPM_DRIVER'] and not ENV['WINDOWS_DDNPM_DRIVER'].empty?
-        ## don't bother with system probe build on x86.
-        command "invoke -e system-probe.build --windows"
-        copy 'bin/system-probe/system-probe.exe', "#{install_dir}/bin/agent"
-      end
+    if ENV['WINDOWS_DDNPM_DRIVER'] and not ENV['WINDOWS_DDNPM_DRIVER'].empty?
+      ## don't bother with system probe build on x86.
+      command "invoke -e system-probe.build --windows"
+      copy 'bin/system-probe/system-probe.exe', "#{install_dir}/bin/agent"
     end
```

Comment:
I think you need to remove this `end` as well, but you might as well remove the entire `if windows_target?` block and invert the following logic, since there's nothing to be done for windows anymore

Reply:
Nice catch ! Fixed in [2d8b2b9](https://github.com/DataDog/datadog-agent/pull/21221/commits/2d8b2b981ce8f8a70ace5fbf3f451da44e9be46e).